store/storetest/plugin_store.go

```diff
@@ -76,7 +76,7 @@ func doTestPluginSaveOrUpdate(t *testing.T, ss store.Store, s SqlSupplier, doer
 			ExpireAt: 0,
 		}
 
-		kv, err := ss.Plugin().SaveOrUpdate(kv)
+		kv, err := doer(kv)
```

Comment:
I had abstracted out `doTestPluginSaveOrUpdate` to allow for testing with both `SaveOrUpdate` and `SetWithOptions`, but never actually /used/ that, so coverage of this path with `SetWithOptions` was effectively missing.